### PR TITLE
DEVDOCS-3390 Clarify order schema

### PR DESCRIPTION
**Pull request by bc-andreadao**
[JIRA Ticket DEVDOCS-3390](https://jira.bigcommerce.com/browse/DEVDOCS-3390) 

#What was asked:
On the [Update an Order](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/updateanorder) page, clarify product_options.id key in the schema to state that this value should be the product_option_id value, instead of the id for the option returned from a GET /order/{id}/products API request.

#What I did:
-Added a description to product_options.id (for the [Update an Order](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/updateanorder) schema, PUT request)
-Added the (same) description to product_option_id (for the [List all Products](https://developer.bigcommerce.com/api-reference/store-management/orders/order-products/getallorderproducts) schema, GET request)
-Referenced/linked the GET request description in the PUT request description

### CODE REVIEW

File: reference/orders.v2.oas2.yml [hunk 1]
```diff
@@ -4532,6 +4532,7 @@ components:
             properties:
               id:
                 type: integer
+                description: 'Numeric ID of an option applied to the product from a list of options available to the product. This field has the same value as `product_option_id` when [retrieving products in an order](https://developer.bigcommerce.com/api-reference/store-management/orders/order-products/getallorderproducts).  '
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:4535]: Double-check the spacing at the end of the sentence and make sure the apostrophes are needed here as line 3762 doesn't call for them.

File: reference/orders.v2.oas2.yml [hunk 2]
```diff
@@ -3759,7 +3759,7 @@ components:
           example: 53
           type: integer
         product_option_id:
-          description: ''
+          description: Numeric ID of an option applied to the product from a list of options available to the product
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:3762]: Period at the end of the sentence here.

File: reference/orders.v2.oas2.yml [hunk 3]
```diff
@@ -97,7 +97,7 @@ paths:
       description: |-
         Updates an *Order*. 
 
-        To add a product to an existing order, don't include `id` in the body. Include `product_options` if adding a product with variants.
+        To add a product to an existing order, don't include `id` in the body. Include `product_options` if adding a product with variants.
```
Comment (bc-tgomez) [inline reference/orders.v2.oas2.yml:100]: Double-check the end of the sentence for extra spacing.

**Review by bc-tgomez — COMMENTED** [review]
Made small edits to the changes to incorporate at your discretion, other than those edits, looks good! Will approve once comments are handled.

File: reference/orders.v2.oas2.yml (hunk 1 above)
Comment (bc-andreadao) [reply reference/orders.v2.oas2.yml]: I got rid of the extra space at the end, but the single quotes are still there. any suggestions?

Comment (bc-andreadao) [reply reference/orders.v2.oas2.yml]: ^Fixed now

**Review by bc-tgomez — APPROVED** [review]
Looks great! 💯

**Pull request merged**